src/modules/status.rs

```diff
@@ -36,6 +51,10 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 })
                 .map(|variable| match variable {
                     "status" => Some(Ok(exit_code)),
+                    "status_int" => Some(Ok(exit_code)),
```

Comment:
What's the differnce between `status_int` and `status` supposed to be?

Reply:
`status` exist for backward compatibility